src/siphon/cdmr/ncstream.py

```diff
@@ -87,8 +84,7 @@ def read_ncstream_data(fobj):
             magic = read_magic(fobj)
         return data, blocks
     else:
-        raise NotImplementedError("Don't know how to handle data type: {0}".format(
-            data.dataType))
+        raise NotImplementedError(f"Don't know how to handle data type: {data.dataType}")
```

Comment:
```suggestion
        raise NotImplementedError(f'Don't know how to handle data type: {data.dataType}')
```
Random double quotes? Didn't check for any more of these.

Reply:
Confused me for a sec just now, but the `'` in "Don't" allows this as double to avoid escaping it.

Reply:
Haha, totally missed it! Even with Github syntax highlighting pointing it out to me.